pandas/tests/generic/test_generic.py

```diff
@@ -339,6 +339,11 @@ def test_copy_and_deepcopy(self, shape, func):
         assert obj_copy is not obj
         self._compare(obj_copy, obj)
 
+    def test_where(self):
+        s = Series(range(5))
+        with tm.assert_produces_warning(FutureWarning):
+            s.where(s > 1, 10, False)
+
```

Comment:
can you put the test in `pandas/tests/frame/indexing/test_where.py` and `pandas/tests/series/indexing/test_where.py`?

Reply:
will do this later today. Getting some flake8 errors and I will learn and fix it.